### PR TITLE
progress bar in taskbar

### PR DESCRIPTION
**Pull request by whyboris**
📽 yeah!

Closes https://github.com/whyboris/Video-Hub-App/issues/45

### CODE REVIEW

File: main-support.ts [hunk 1]
```diff
@@ -682,4 +682,9 @@ export function updateFinalArrayWithHD(
  */
 export function sendCurrentProgress(current: number, total: number, stage: number): void {
   globals.angularApp.sender.send('processingProgress', current, total, stage);
+  if (stage !== 0) {
+    globals.winRef.setProgressBar(current / total);
```
[the remark this answers is not in the document]
Comment (whyboris) [reply main-support.ts]: You're correct! They lied to me 😝 https://electronjs.org/docs/tutorial/progress-bar
> setting the parameter to a value below zero (like -1) will remove the progress bar

I tested on Windows and it worked, but will test on Mac now to see how to make it disappear 👍

**Comment by whyboris** [comment]
This is an excellent reference: https://blog.dcpos.ch/how-to-make-your-electron-app-sexy

But here too it says to have _setProgressBar_ to `-1`

**Comment by cal2195** [comment]
The problem is you never set it to -1!

**Comment by cal2195** [comment]
Or maybe you do..

**Comment by cal2195** [comment]
Line 813 main.ts 😉

**Pull request merged**